handsontable/test/types/settings.types.ts

```diff
@@ -448,6 +448,7 @@ const allSettings: Required<Handsontable.GridSettings> = {
   afterSelectionEnd: (r, c, r2, c2, selectionLayerLevel) => {},
   afterSelectionEndByProp: (r, p, r2, p2, selectionLayerLevel) => {},
```

Comment:
Thanks for updating the TS tests. However, can I ask you to add new methods tests to the `handsontable/test/types/methods.types.ts` file? There is no cover for `setData` and `updateData` methods.